appveyor.yml

```diff
@@ -33,3 +33,7 @@ artifacts:
     name: OS-X minishift binary
   - path: out\windows-amd64\minishift.exe
     name: Windows minishift binary
+
+notifications:
+  - provider: Webhook
+    url: http://minibot.34e99f76.svc.dockerapp.io:9009/hubot/appveyor
```

Comment:
I need to make some more changes on the minot side and re-deploy, but this way the hook is already enabled :-)